SWDEV-439933 - Use relative path in hipfc script

raramakr opened the pull request:
Absolute paths are used in hipfc scripts for hipfort compiler and hipfort install directory Use relative paths for the same

raramakr committed:
SWDEV-439933 - Use relative path in hipfc script

Absolute paths are used in hipfc scripts for hipfort compiler and hipfort install directory
Use relative paths for the same

cgmb reviewed (approved):
LGTM, although it may take a few days before the CI is working again.

raramakr committed:
Correct the hipfort compiler relative path in Makefile.hipfort

The relative path of Makefile.hipfort should be calculated with respect to the install path of the file

raramakr committed:
Correct the relative path generation in hipfc

hipfc can be invoked from any directory, since update-alternatives is run for hipfc binary. Invoking hipfc from a directory other than /opt/rocm-ver/bin was providing incorrect relative paths. Corrected the same

raramakr commented:
Tested the patch by running the following in hipFORT/test directory. All tests passed
 make run_all
@cgmb  Can you pls review the latest patch
